Documentation/network/servicemesh/gateway-api/host-network-mode.rst

```diff
@@ -0,0 +1,103 @@
+
+Host network mode
+#################
+
+.. note::
+    Supported since Cilium 1.16+
+
+This mode allows you to expose the Cilium Gateway API Gateway directly on the
+host network.
+This is useful in some edge cases where one doesn't want, or can't, use
+LoadBalancer type services (dev environments, cluster-external loadbalancer,
+...).
+
+.. note::
+    * Enabling the Cilium Gateway API host network mode automatically disables the LoadBalancer type service mode. They are mutually exclusive.
+    * The listener is exposed on all interfaces (``0.0.0.0`` for IPv4 and/or ``::`` for IPv6).
+
+The host network mode can be enabled via Helm flag:
+
+.. code-block:: yaml
+
+    gatewayAPI:
+      enabled: true
+      hostNetwork:
+        enabled: true
+
+Once enabled, the host network ports can be specified on the ``Gateway``
+resources via ``spec.listeners.port``. The port must be unique per ``Gateway``
+resource and you should choose a port number higher than 1023 (see
+`Bind to privileged port`_).
+
+.. warning::
+    Be aware that misconfiguration might result in port clashes. Configure unique ports that are still available on all Cilium Nodes where Gateway API listeners are exposed.
+
+Bind to privileged port
+=======================
+
+By default, it's mandatory to use ports higher than ``1023`` as the Cilium L7
+Envoy process does not have any Linux capabilities out-of-the-box (so no
+``CAP_NET_BIND_SERVICE``) and is therefore not allowed to listen on privileged
+ports.
+If you choose a port equal to or lower than ``1023``, ensure that
+``envoy.securityContext.capabilities.keepCapNetBindService=true`` is configured
+and to add the capability ``NET_BIND_SERVICE`` to the respective
+:ref:`Cilium Envoy container via Helm values<envoy>`:
+
+* Standalone DaemonSet mode: ``envoy.securityContext.capabilities.envoy``
+* Embedded mode: ``securityContext.capabilities.ciliumAgent``
+
+Configure the following to allow privileged port bindings in host network mode:
+
+.. code-block:: yaml
+
+    gatewayAPI:
+      enabled: true
+      hostNetwork:
+        enabled: true
+    # For Cilium Envoy standalone DaemonSet mode:
+    envoy:
+      enabled: true
+      securityContext:
+        capabilities:
+          keepCapNetBindService: true
+          envoy:
+          # Add NET_BIND_SERVICE to the list (keep the others!)
+          - NET_BIND_SERVICE
+    # For Cilium Envoy embedded mode:
+    envoy:
+      securityContext:
+        capabilities:
+          keepCapNetBindService: true
+    securityContext:
+      capabilities:
+        ciliumAgent:
+        # Add NET_BIND_SERVICE to the list (keep the others!)
+        - NET_BIND_SERVICE
+
+Deploy Gateway API listeners on subset of nodes
+===============================================
+
+It's supported to expose the Cilium Gateway API Envoy listener only on specific
+nodes. This only works in combination with the host network mode and can be
+configured via a node label selector in the Helm values.
```

Comment:
Couple of nits:

1. Use a colon to transition to the helm values.
2. In the first sentence, stating that the functionality "is supported" is a bit redundant, since we can just say what is supported outright:

```suggestion
The Cilium Gateway API Envoy listener can be exposed on a specific subset of
nodes. This only works in combination with the host network mode and can be
configured via a node label selector in the Helm values:
```